app/Jobs/UpdateVestaOpeninghours.php

```diff
@@ -46,8 +56,10 @@ public function handle()
 
         try {
             $openinghoursService = app('OpeninghoursService');
-            $openinghoursService->isOpenForFullWeek();
-            $output = $formatter->render('html', $openinghoursService->getData());
+            $start = (new Carbon())->startOfWeek();
+            $end = (new Carbon())->endOfWeek();
+            $openinghoursService->collectData($start, $end, Service::find($this->serviceId));
+            $output = $this->formatter->render($openinghoursService->getData());
```

Comment:
Output will be collected as:
$this->formatter->render($openinghoursService->getData());
$output = $this->formatter->getOutput();